core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/components/dot-experiments-reports-chart/dot-experiments-reports-chart.component.html

```diff
@@ -6,7 +6,7 @@ <h1 class="header-title" data-testId="chart-title">
             </h1>
             <div id="legend-container" data-testId="chart-legends"></div>
         </div>
-        <ng-container *ngIf="!isEmpty; else emptyStateTpl">
+        <ng-container *ngIf="!isEmpty && data; else emptyStateTpl">
```

Comment:
Can't you just use `data`?

Reply:
Removed data, `isEmpty` comes from a selector checking the result if is 0 `results?.sessions.total === 0`